virttest/qemu_vm.py

```diff
@@ -586,7 +586,7 @@ def add_log_anaconda(devices, pci_bus='pci.0'):
 
         def add_smp(devices):
             smp_str = " -smp %d" % self.cpuinfo.smp
-            smp_pattern = "smp .*\[,maxcpus=cpus\].*"
+            smp_pattern = "smp .*\[,maxcpus=.*\].*"
```

Comment:
How about `"smp .*\[,maxcpus=\w*cpus\].*"`

Reply:
Yes, both are OK. But `.*\]` may leak to other properties' namespace.
BTW, for the easiest way to find maxcpus support, it can be `smp .*maxcpus=.*`? If we ignore all risks.